components/availability/src/Availability.svelte

```diff
@@ -247,6 +253,19 @@
           availability = AvailabilityStatus.BUSY;
         }
 
+        if (
+          availability === AvailabilityStatus.PARTIAL &&
+          required_participants.length
+        ) {
+          if (
+            !required_participants.every((participant) =>
+              freeCalendars.includes(participant),
```

Comment:
After thinking about it, could this be made into a variable for readability?

Reply:
Got it; setting up as this:

```
  const allRequiredParticipantsIncluded = (calendars: string[]) => {
    return required_participants.every((participant) => calendars.includes(participant));
  }

...................
          if (!allRequiredParticipantsIncluded(freeCalendars)) {...
```